src/ember-app.js

```diff
@@ -450,6 +450,8 @@ function createShoebox(doc, fastbootInfo) {
 
     let value = shoebox[key];
     let textValue = JSON.stringify(value);
+    if (textValue === undefined) { continue; }
```

Comment:
@rwjblue That is unnecessary.

> After verifying that shoebox.retrieve returns undefined if the key doesn't exist in ember-fastboot/ember-cli-fastboot#406, it doesn't make much sense to jam undefined in the shoebox at all. So this now ignores undefined instead.

We don't need to put anything in the payload because the desired behavior already works on the other end.